README.md

```diff
@@ -1,218 +1,180 @@
 
-```sh
-./ankerctl.py webserver run
-```
+Additional instructions can be found in the web interface.
+
+![Screenshot of prusa slicer](/static/img/setup/prusaslicer-2.png "Screenshot of prusa slicer")
 
-You can alternativly use docker compose to start the webserver running behind nginx
+### Command-line tools
+
+Some examples:
 
 ```sh
-docker compose up
-```
+# run the webserver to control over webgui
+./ankerctl.py webserver run
 
+# attempt to detect printers on local network
+./ankerctl.py pppp lan-search
 
+# monitor mqtt events
+./ankerctl.py mqtt monitor
 
-## Docker
+# start gcode prompt
+./ankerctl.py mqtt gcode
 
-While running the python script is generally prefered, there may be situations where you want a more portable solution. For this, a docker image is provided.
+# set printer name
+./ankerctl.py mqtt rename-printer BoatyMcBoatFace
 
-```sh
-docker build -t ankerctl .
-```
+# print boaty.gcode
+./ankerctl.py pppp print-file boaty.gcode
 
-Example usage (no peristent storage)
-```bash
-docker run \
-  -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-  ankerctl config decode /tmp/login.json
-```
+# capture 4mb of video from camera
+./ankerctl.py pppp capture-video -m 4mb output.h264
 
-Example usage (with peristent storage)
-```bash
-# create volume where we can store configs
-docker volume create ankerctl_vol
-
-# generate /root/.config/ankerctl/default.json which is mounted to the docker volume
-docker run \
-  -v ankerctl_vol:/root/.config/ankerctl \
-  -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-  ankerctl config import /tmp/login.json
-
-# Now that there is a /root/.config/ankerctl/default.json file that persists in the docker volume
-# we can run ankerctl without having to specify the login.json file
-docker run \
-  -v ankerctl_vol:/root/.config/ankerctl \
-  ankerctl config show
+# select printer to use when you have multiple
+./ankerctl.py -p <index> # index starts at 0 and goes up to the number of printers you have
 ```
 
-
 ## Legal
 
-This project is in ABSOLUTELY NO WAY endorsed, affiliated with, or supported by
-Anker. All information found herein is gathered entirely from reverse
-engineering.
+This project is **<u>NOT</u>** endorsed, affiliated with, or supported by AnkerMake. All information found herein is gathered entirely from reverse engineering using publicly available knowledge and resources.
 
-The goal of this project is to make the Ankermake M5 usable and accessible using
-only Free and Open Source Software.
+The goal of this project is to make the AnkerMake M5 usable and accessible using only Free and Open Source Software (FOSS).
 
-This project is [licensed under the GNU GPLv3](LICENSE), and copyright © 2023
-Christian Iversen.
+This project is [licensed under the GNU GPLv3](LICENSE), and copyright © 2023 Christian Iversen.
```

Comment:
We should add image attributions here for the SVGs that we are using

Reply:
Oh yes, good point. In fact, we have to. @billyjbryant do you have a link? or who added them originally?

Reply:
@chrivers didn't you collect this somewhere?

Reply:
v1.1 item, will add ticket